### ISSUE TITLE
docs(tutorial) - toh-pt2 could be more clear in the "Expose heroes" section

### ISSUE THREAD

**Issue by dogeared** [angular/angular#19714]
## I'm submitting a...

<pre><code>
[ ] Regression (a behavior that used to work and stopped working in a new release)
[ ] Bug report  
[ ] Feature request
[x] Documentation issue or request
[ ] Support request => Please do not submit support request here, instead see https://github.com/angular/angular/blob/master/CONTRIBUTING.md#question
</code></pre>

## Current behavior

For n00bs, in part 2 of the tutorial, it is not clear where the `heroes = HEROES;` line should go exactly.

## What is the motivation / use case for changing the behavior?

To make it easier for newcomers to angular to follow the tutorial

## Environment

<pre><code>
Angular version: 4.4.5

Browser:
all
 
</code></pre>

**Comment by trotyl** [comment]
The order of properties does not matter at all in any kind of class, why should it be "cleared" if it's not a requirement?

**Comment by dogeared** [comment]
Not that the order was important but where in the file the

heroes = HEROES

should go.

As an angular n00b, it was not clear to me.

**Comment by trotyl** [comment]
> Create a public property ...

Angular tutorial assume that you already have JavaScript background, this is not a JavaScript tutorial.

**Comment by dogeared** [comment]
I have a deep JavaScript background. ;)

It's fine if you don't want to accept. IMHO it could be made more clear
where that statement is supposed to go.

**Comment by kapunahelewong** [comment]
Hey @dogeared thanks for bringing this up. I agree – clarifying this can only help improve the learning experience. I'll make sure to pass this along to the right person on the docs team.

**Comment by dogeared** [comment]
Thanks! There are other places in the tutorial where it tells you to put
something new between `this` and `that`, even though the order isn't
necessarily important. It just makes it more clear.

==============================================
Micah Silverman, CSM <http://bit.ly/WikiScrum>
Phone: 631.606.8928
Co-Author: Mastering Enterprise Javabeans 3.0 <http://bit.ly/MasteringEJB3>
==============================================
"Debugging is twice as hard as writing the code in the first
place. Therefore, if you write the code as cleverly as
possible, you are, by definition, not smart enough to debug
it." by Brian Kernighan
==============================================
[image: Namez]
<http://namez.com/profiles/1720-micah-silverman/?autoPlay=true>

**Comment by kapunahelewong** [comment]
Yeah, I know exactly what you mean. Feel free to add any more you notice here or in your PR and to tag me when you do. I've stumbled over some like that and try to fix them when I see them, but haven't done a whole sweep. The more eyes, the better. Thanks again for catching that.

**Comment by tbosch** [comment]
Would be nice if somebody put a PR together.

**Comment by dogeared** [comment]
er... I did. https://github.com/angular/angular/pull/19715

**Comment by chembu** [comment]
PR #19715
